<h4>Problem statement</h4>
Sales of products in four different regions is tabulated for males and females. Find if male-female buyer rations are similar across regions.

In [40]:
import pandas as pd
import numpy as np

In [41]:
df = pd.read_csv('/content/drive/My Drive/DS/Assignments/HypothesisTesting/BuyerRatio.csv')
df.head()

Observed Values  East  West  North  South
0           Males    50   142    131     70
1         Females   435  1523   1356    750

Inputs given are 4 discrete variables(east,west,north,south).

Output needed is also discrete. We are trying to find out if proportions of male and female are similar or not across the regions.

We will proceed with <Strong>chi-square test</Strong>.

According to problem statement, the hypothesis can be formed as:<br>
H<sub>o</sub> => Proportions of Male and Female are similar.<br>
H<sub>A</sub> => Proportions of Male and Female are not similar.<br>
We consider α = 0.05 (Significance level).


In [56]:
from scipy import stats

stat, pvalue, dof, epectedfreq = stats.chi2_contingency([df["East"], df["West"], df["North"], df["South"]])
print("P-value: {}".format(pvalue))

P-value: 0.6603094907091882


The p-value 0.66 is higher than the significance value. <br>
<strong> Hence we accept the null hypothesis and conclude that the Male and Female proportions are similar</strong> 